build_apk.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# CI script for building and deploying Editions android APK
```

Comment:
Would suggest adding [`set -e`](https://explainshell.com/explain?cmd=set+-e) to ensure we exit on the first error.

Reply:
Thanks. Now added